lib/sidekiq_flow/client.rb

```diff
@@ -1,39 +1,42 @@
 module SidekiqFlow
   class Client
-
     class << self
+      attr_accessor :adapters
```

Comment:
This would probably just be attr_writer (setter) as we already have the private method adapters (getter).